src/crawlee/statistics/models.py

```diff
@@ -27,6 +29,20 @@ class FinalStatistics:
     requests_total: int
     crawler_runtime: timedelta
 
+    def to_table(self) -> None:
+        """Print out the Final Statistics data as a table."""
+        table = Table()
+        table.add_column('INFO')
+        table.add_column('Statistics')
```

Comment:
```suggestion
        table = Table(show_header=False)
        table.add_column()
        table.add_column()
```

The headers are not particularly useful in this case.